tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -492,13 +490,6 @@ def test_no_feature(self):
         assert resp.status_code == 404
 
     def test_no_perms(self):
-        # Downgrade user from "owner" to "member".
-        with unguarded_write(using=router.db_for_write(OrganizationMember)):
-            OrganizationMember.objects.filter(user_id=self.user.id).update(role="member")
-
-        resp = self.get_response(self.organization.slug)
-        assert resp.status_code == 403
-
```

Comment:
Might make sense to keep this section, but validate that the response is successful until we turn off the option